lib/mayaUsd/utils/utilFileSystem.h

```diff
@@ -34,6 +34,17 @@ std::string resolvePath(const std::string& filePath);
 MAYAUSD_CORE_PUBLIC
 std::string getDir(const std::string& fullFilePath);
 
+/*! \brief takes in two absolute file paths and returns the first one to second one.
+           Also return a boolean: true if the path was successfully made relative,
+           false otherwise.
+
+           If the second path is not absolute or is not reachable from the first,
+           then the returned path will still be absolute.
+ */
```

Comment:
We need to change slightly the behaviour of this function and document it, in order for it to correspond to behaviour of getPathRelativeToMayaSceneFile, where it is written: "When there is no scene file, the absolute (input) path will be returned."